website/docs/r/images_ims_v2.html.markdown

```diff
@@ -0,0 +1,97 @@
+---
+layout: "opentelekomcloud"
+page_title: "OpenTelekomCloud: opentelekomcloud_images_ims_v2"
+sidebar_current: "docs-opentelekomcloud-resource-images-ims-v2"
+description: |-
+  Manages a V2 Image resource within OpenTelekomCloud IMS.
+---
+
+# opentelekomcloud\_images\_ims_v2
+
+Manages a V2 Image resource within OpenTelekomCloud IMS.
+
+## Example Usage
```

Comment:
better to also give an example with OBS here.

Reply:
@niuzhenguo Thanks for your comments, please review it in the new commits.